### ISSUE TITLE
Statusbar elapsed time gets erroneously updated even after statusbar is closed

### ISSUE THREAD

**Issue by guano** [Rockhopper-Technologies/enlighten#66]
**Describe the bug**
A statusbar's `elapsed` field gets updated, even if it is closed and not refreshed.

Looks like this bug was fixed for counters but not statusbars
see issue #62 

**To Reproduce**
``` python
import time                                                                                                   
import enlighten                                                                                              
                                                                                                              
enlightenmanager=enlighten.get_manager()                                                                      
                                                                                                              
sb = enlightenmanager.status_bar(                                               
            leave=True, autorefresh=False, color="black_on_green",                                            
            status_format=u"statusbar elapsed:{elapsed}")                                                     
time.sleep(1)                                                                                                 
sb.update()                                                                                                   
time.sleep(1)                                                                                                 
sb.update()                                                                                                   
time.sleep(1)                                                                                                 
sb.update()                                                                                                   
time.sleep(1)                                                                                                 
                                                                                                              
sb.color = "black_on_red"                                                                                     
sb.update()                                                                                                   
sb.close()                                                                                                    
time.sleep(5)                                                                                                 
                                                                                                              
sb2 = enlightenmanager.status_bar(                                                                            
            leave=True, autorefresh=False, color="black_on_green",                                            
            status_format=u"statusbar2 {elapsed}")                                                            
time.sleep(1)                                                                                                 
sb2.close()                                                                                                   
                                                                                                              
enlightenmanager.stop()     
```

sb's elapsed time gets updated even though it's not being updated and is in fact closed.

**Environment (please complete the following infkkkormation):**
 - Enlighten Version: commit 91e39991b655462c4553e5266bad365abda5c63c (`master` as of May 31, 2024)
 - OS and version: CentOS Linux 7
 - Console application: tmux
 - Special Conditions: -

**Comment by avylove** [comment]
Thanks for reporting! I left some comments on the PR, but I think the main thing to sort out is when to determine the clock should be stopped for status bars. You went with the closed time which does make sense in some cases, but there could be an argument for the last update in other cases. I'm not sure what the right answer is. I'm leaning towards agreeing with you, but my hesitation is this is different than the way it's handled in progress bars. That may be ok, but I think the "why" needs to be understood and documented.

**Comment by guano** [comment]
Yeah, it definitely is a decision that can be disputed either way.

In the meantime I have figured out a workaround for my use case at least. The `elapsed` variable can just be read at close time and inserted as a static string rather than being dynamically updated whenever the manager adds/removes another bar. 

```python
from enlighten._util import format_time

# start, sleep, update, etc as in original code sample

sb.update(f"statusbar finished. elapsed: {format_time(sb.elapsed)}")
sb.close()
```

**Comment by avylove** [comment]
Fixed in [1.13.0](https://github.com/Rockhopper-Technologies/enlighten/releases/tag/1.13.0)
I ended up taking going with stopping the clock when closed, but just changed ``_closed`` to a float instead of a Boolean rather than introducing another attribute. Thanks for reporting. Sorry this took so long. Please try it out and let me know if you have any issues.

**Comment by guano** [comment]
That is a nice clean fix, and I can confirm it works great. Thank you.
